app/models/policy.rb

```diff
@@ -4,6 +4,8 @@ class Policy < ApplicationRecord
   belongs_to :policy_class
   has_one :comment, dependent: :destroy
 
+  default_scope { order(:section) }
```

Comment:
Ensure that the policies are always displayed '1a, 1b, 2a' etc. 

I tried putting the ordering in `PolicyClass` (`has_many :policies, -> { order(:section) }`) but this seems to cause errors when used in conjunction with `accepts_nested_attributes_for`. I tried looking for known bugs without much success 😬